etc/ATSCam/primary_hdu.header

```diff
@@ -63,8 +80,8 @@ GROUPID =                      /
         ---- Information from Camera
 CCD_MANU= 'ITL'                / CCD Manufacturer
 CCD_TYPE= '3800C'              / CCD Model Number
-CCD_SERN= '20304'              / Manufacturers? CCD Serial Number
-LSST_NUM= 'ITL-3800C-098'      / LSST Assigned CCD Number
+CCD_SERN= '20862'              / Manufacturers? CCD Serial Number
+LSST_NUM= 'ITL-3800C-068'      / LSST Assigned CCD Number
```

Comment:
Two problems:  why is this change in a WCS ticket, and why are we hard-coding this information in the header service?  The value is known in cameraGeom and presumably to the camera;  if you are not getting it from there please omit it.

Reply:
LSE-400 requires it be there. Also, CCS files seem to include it so the camera has some idea of the LSST_NUM. I agree that hard-coding it in the header service seems wrong if CCS already knows about it. We do want the information in there though since it's the only place in the files that tells us which CCD is really being used (granted, not if it's hard-coded here). All LSST data files have always had LSST_NUM specified.

Reply:
OK, and I agree that it's good to have it.  However, hard-coding the value is worse than useless, so please get it from the CCS.  If this isn't possible, please open a CAP ticket and we can discuss solutions.

Reply:
According to Tony J these should not change (unless something really bad happens). However, I have already requested Tony to publish these as `ATCamera` events. These will be available via the `ATCamera_logevent_wrebSettingsApplied` event on the next tag of `ts_xml`, which should be probably release this week.

Reply:
The `master` branch of `ATHeaderService` now has the above changes as part of the `ts_xml 4.3.0` updates on commit `25c2ca1`

Reply:
I'm confused.  Your comment says that the `ts_xml 4.3.0` is broadcasting the CCD serial number, but you're still hard-coding it here.  Also, could you explain why this change is in a WCS ticket?

Reply:
I've merger `master` into `tickets/DM-19669` to reflect the changes that were made a while on the main branch. The values are no longer hard coded.